phoenicis-javafx/src/main/resources/javafx.properties

```diff
@@ -25,4 +25,5 @@ application.windowMaximized                 =           false
 application.appsListType                    =           ICONS_LIST
 application.containersListType              =           ICONS_LIST
 application.enginesListType                 =           ICONS_LIST
-application.libraryListType                 =           ICONS_LIST
\ No newline at end of file
+application.libraryListType                 =           ICONS_LIST
+application.fuzzySearchRatio                =           85
```

Comment:
I think we should add some comments to this file, to make the content more easy to understand for the users or other developers.